### ISSUE TITLE
Segmentation Fault when make run-example-c-project-verilator

### ISSUE THREAD

**Issue by vanphuc1208** [openhwgroup/cva5#28]
While running the simulation, I encountered an issue that I haven't been able to resolve. Specifically, when I run  make run-example-c-project-verilator (or the other tests), they all result in the segmentation error (image below).
![error](https://github.com/user-attachments/assets/21413091-dd58-4837-8325-aab4b08fd51b)

**Comment by CKeilbar** [comment]
Which version of Verilator are you running?
I believe they changed something relatively recently that caused this; I suggest applying this patch: https://github.com/CKeilbar/ck-cva5/commit/33a2546033779d681f84ec31732168c19562d96f

**Comment by MikeOpenHWGroup** [comment]
Hi @vanphuc1208, did @CKeilbar answer your question?  If so, please acknowledge and close this issue.  Thanks!

**Comment by vanphuc1208** [comment]
Yeah! I have solved this problem by changing the Verilator version. Thanks for your support.
